tests/unit/test_endpoints.py

```diff
@@ -102,5 +110,14 @@ def test_get_async_query_status_when_job_is_complete(self, mock_s3, mock_job_sta
             'presigned_url': 'www.ThisUrlShouldWork.com'})
 
 
+@contextmanager
+def manage_query_timeout(timeout_seconds):
+    resource = config.reset_db_timeout_seconds(timeout_seconds)
```

Comment:
- "resource" should be named something more specific
- Since you're not using the resource at all in this case (your `with` clause does not bind it as a variable like `with foo() as bar:`), and since `config.reset_db_timeout_seconds()` returns nothing, you can just omit `resource` and `yield` to the caller without a value.

Reply:
👍